packages/icons/src/create-pr.svg

```diff
@@ -0,0 +1 @@
+<svg width="26" height="23" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M2.786 1.95c-.513 0-.929.428-.929.956v17.223c0 .529.416.957.929.957h20.428c.513 0 .929-.428.929-.957V7.69c0-.528.416-.957.928-.957.513 0 .929.429.929.957v12.44c0 1.585-1.247 2.87-2.786 2.87H2.786C1.247 23 0 21.715 0 20.13V2.905C0 1.321 1.247.036 2.786.036h16.25c.512 0 .928.428.928.956 0 .529-.415.957-.928.957H2.786zm3.882 5.192a.911.911 0 011.293-.236l4.371 3.154c.66.476.66 1.483 0 1.96l-4.37 3.152a.911.911 0 01-1.294-.235.975.975 0 01.228-1.332l3.556-2.566-3.556-2.565a.975.975 0 01-.228-1.332zm8.189 6.29c-.513 0-.929.428-.929.956 0 .529.416.957.929.957H19.5c.512 0 .928-.428.928-.957 0-.528-.416-.957-.928-.957h-4.643z" fill="#0278D5"/></svg>
```

Comment:
pls remove height/width and use viewbox instead. pls replace hex fill color with `currentColor`.
 
 see SVG guidelines here: https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices